lib/settings.rb

```diff
@@ -72,7 +72,8 @@ def scrum_defaults
       },
       'label_names' => {
         'sticky' => 'Sticky',
-        'waterline' => 'Under waterline'
+        'waterline' => 'Under waterline',
+        'in_last_sprint' => 'In Last Sprint'
```

Comment:
To be consistent with the other labels I think we shouldn't capitalise every word, so: `'In last sprint'`